engine/src/cmd/collision.cpp

```diff
@@ -150,18 +150,9 @@ void Collision::dealDamage(Collision other_collision, double deltaKE_linear, dou
         return;
     }
 
-    // Shorten variables for better visibility
-    float& m1 = mass;
-    float& m2 = other_collision.mass;
-    Vector& v1 = velocity;
-    Vector& v2 = other_collision.velocity;
-
-    float relative_velocity = (v1-v2).Magnitude();
-
     Damage damage;
-    damage.normal_damage = relative_velocity * relative_velocity *
-            (m2)/(m1+m2) * factor;
-
+    damage.normal_damage = 0.5 * ( deltaKE_linear + deltaKE_angular) /* deltaKE is in KiloJoules, due to mass being in units of 1000 Kg, not Kg - so convert accordingly */ / configuration.physics.kilojoules_per_damage; // assign half the change in energy to this unit, convert from KJ to VSD
```

Comment:
Would prefer the comment to not be in-line with the equation:
```
// deltaKE = in KiloJoules due to mass being in units of 1,000 Kg, not simply Kg; convert accordingly
// and then assign half the change in energy to convert from KJ to VSD
damage.normal_damage = 0.5 * ( deltaKE_linear + deltaKE_angular) / configuration.physics.kilojoules_per_damage;
```

It's cleaner, easier to read and more  useful.

Reply:
Not actually my commit, but fixed anyway.